docs/conf.py

```diff
@@ -68,3 +68,19 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
+
+
+def run_apidoc(_):
```

Comment:
Do you have to copy paste the url or can it be posted here automatically somehow?